core/src/boot.py

```diff
@@ -31,7 +31,7 @@
                     return
                 else:
                     label = "Wrong PIN, enter again"
-        except (OSError, PinCancelled, SdCardUnavailable) as e:
```

Comment:
@andrewkozlik do you remember where the `OSError` might be coming from here?
ISTM it would have to come from `request_sd_salt`, which should now only raise `SdCardUnavailable` -- hence OSError (or FatFSError) should not be caught, fall through to BaseException and halt the device

Reply:
I think you are correct.

Reply:
i'll remove the `OSError` after the rebase